### ISSUE TITLE
Central location

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11561]
Preferred term label:
Central location

Synonyms

    Centrally located

    Central region

Definition (free text, please give PubMed ID)
An anatomical spatial pattern or localization referring to the middle or center of a structure. In ophthalmology, this term frequently modifies phenotypes to describe central corneal opacities, central visual field scotomas, or central posterior lens opacities. PMID:10532444, PMID:24552665

Parent term (use hpo.jax.org/app)
The anatomical localization of the specified phenotypic abnormality (HP:0012830)

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)

    Macular corneal dystrophy (OMIM:217800)

    Central areolar choroidal dystrophy (OMIM:213000)

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by daboe01** [comment]
sorry for the dupe.
HP:0030645
